apps/overlay-sandbox/app/cases/overlay-manager/page.tsx

```diff
@@ -9,7 +9,7 @@ import { useEffect, useRef } from 'react';
 
 const overlayOptions: Omit<ChatOverlayManagerOptions, 'hostDomain'> = {
   id: 'test',
-  domain: 'http://localhost:3000',
+  domain: process.env.E2E_OVERLAY_HOST ?? 'http://localhost:3000',
```

Comment:
I don't think that E2E overlay host should be insidenormal build - change it to just process.env.OVERLAY_HOST and set it on sandbox app level in env variables if possible

Reply:
thank you, resolved